paramak/utils.py

```diff
@@ -654,6 +654,45 @@ def export_wire_to_html(
     return fig
 
 
+def convert_circle_to_spline(
+        p0: Tuple[float, float],
+        p1: Tuple[float, float],
+        p2: Tuple[float, float],
+        tolerance: Optional[float] = 0.1
+) -> List[Tuple[float, float, str]]:
+    """Converts three points on the edge of a circle into a series of points
+    on the edge of the circle. This is done by creating a circle edge from the
+    the points provided (p0, p1, p2), facets the circle with the provided
+    tolerance to extracts the points on the faceted edge and returns them.
+
+    Args:
+        p0: coordinates of the first point
+        p1: coordinates of the second point
+        p2: coordinates of the third point
+        tolerance: the precision of the faceting.
+
+    Returns:
+        The new points
+    """
+
+    # work plane is arbitrarily selected and has no impact of function
+    solid = cq.Workplane('XZ').center(0, 0)
+    solid = solid.moveTo(p0[0], p0[1]).threePointArc(p1, p2)
+    edge = solid.vals()[0]
+
+    new_edge = paramak.utils._transform_curve(edge, tolerance=tolerance)
+
+    points = paramak.utils.extract_points_from_edges(
+        edges=new_edge,
+        view_plane='XZ'
+    )
+    points_with_connections = []
+    for point in points:
+        points_with_connections.append((point[0], point[1]))
```

Comment:
what's the difference between `points` and `points_with_connections` ?

Reply:
ah  yes, sorry I think they were different early on in this PR but now they are the same. I can remove this.